CHANGELOG.md

```diff
@@ -4,15 +4,26 @@ All notable changes to this project will be documented in this file.
 
 ## [master]
 
+This release mainly removes a technical dept where the GitHub Pull-Requests crawler
+was managed differently than other crawlers. With this release, the crawler is managed
+by the same process (or container) than other crawlers and respects the same Macroscope
```

Comment:
I think we should keep the `Macroscope` and `Lentille` names for the developper documentation, that should not be visible to the end user.

```suggestion
by the same process (or container) than other crawlers and respects the same
```

Reply:
done